index.yaml

```diff
@@ -1,16 +1,5 @@
 indexes:
 
-# AUTOGENERATED
-
-# This index.yaml is automatically updated whenever the dev_appserver
-# detects that a new type of query is run.  If you want to manage the
-# index.yaml file manually, remove the above marker line (the line
-# saying "# AUTOGENERATED").  If you want to manage some indexes
-# manually, move them above the marker line.  The index.yaml file is
-# automatically uploaded to Cloud Datastore when you next deploy
-# your application using 'gcloud app deploy' or create the indexes
-# using 'gcloud datastore indexes create'.
-
```

Comment:
Why is this removed?

Reply:
Actually I didn't remove it explicitly it was added automatically and then in a commit earlier when I was reverting some file. Should I add it again?